pkg/playground/react-patterns.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
     <script src="../base1/bundle.js"></script>
-    <script src="../base1/cockpit-components.js"></script>
+    <script src="../performance/bundle.js"></script>
```

Comment:
Also so that we don't load dialog-view.js when the bundle hasn't been loaded yet.